src/utilities/TransactionUtilities.ts

```diff
@@ -85,13 +86,15 @@ export const mapTransfer = (transaction: Transaction): CacheTransferTransaction
  * @param {TransferTransaction} transaction
  * @returns {boolean}
  */
-export const cacheDetails = (transaction: TransferTransaction): CACHE => {
-  if (transaction.containsMosaics()) {
-    const cache = transaction.mosaics().find(mosaic => mosaic.mosaicId.namespaceId === CACHE.MOSAICID.namespaceId && mosaic.mosaicId.name === CACHE.MOSAICID.name);
-    if (cache) { return new CACHE(cache.quantity / Math.pow(10, CACHE.DIVISIBILITY)); }
-  }
-  return new CACHE(0);
-};
+export const mosaicDetails = (transaction: TransferTransaction): MosaicTransferable | CACHE => {
```

Comment:
So i dont understand why this function and the others are in a file called TransactionUtilities? Is this the best place for them? Wouldn't it be better to Mosaic.mosaicDetails() or something?